client/common/directives/navigation/navigation.controller.coffee

```diff
@@ -1,15 +1,13 @@
 navigationController = ($meteor) ->
     vm = this
 
-    if vm.sectionId?
+    if !!vm.sectionId
         $meteor.call('getSectionById', vm.sectionId).then ->
-            # TODO użyć filtrów kiedy zmerguje się ich gałąź
-            vm.section = Sections.findOne({ id: vm.sectionId })
+            vm.section = Filter.Sections.by(vm.sectionId)
 
-    if vm.topicId?
+    if !!vm.topicId
```

Comment:
sprawdzałeś to? jeżeli sectionId = '' wtedy sectionId? daje true... najlepiej używać podwójnych wykrzykników